fiftyone/core/stages.py

```
@@ -8768,8 +8827,18 @@ def load_view(self, sample_collection):
         else:
             name = None
 
-        if state != last_state or not fod.dataset_exists(name):
-            kwargs = self._config or {}
+        try:
+            last_dataset = fod.load_dataset(name)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)